<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ViewSensors</class>
 <widget class="QWidget" name="ViewSensors">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>506</width>
    <height>343</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RBE</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="3" column="1">
    <widget class="QLabel" name="sensor1">
     <property name="text">
      <string>98,0  Bar</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="labelS2">
     <property name="text">
      <string>Sensor 2</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="labelS1">
     <property name="text">
      <string>Sensor 1</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="labelS5">
     <property name="text">
      <string>Sensor 4</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QProgressBar" name="progressBarSystem">
     <property name="value">
      <number>0</number>
     </property>
    </widget>
   </item>
   <item row="5" column="2">
    <widget class="QProgressBar" name="progressBarS3">
     <property name="value">
      <number>0</number>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="labelS4">
     <property name="text">
      <string>Sensor 3</string>
     </property>
    </widget>
   </item>
   <item row="6" column="2">
    <widget class="QProgressBar" name="progressBarS4">
     <property name="value">
      <number>0</number>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QLabel" name="sensor3">
     <property name="text">
      <string>101,4 Bar</string>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QProgressBar" name="progressBarS2">
     <property name="value">
      <number>0</number>
     </property>
     <property name="textVisible">
      <bool>true</bool>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="invertedAppearance">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QProgressBar" name="progressBarS1">
     <property name="value">
      <number>0</number>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QLabel" name="sensor2">
     <property name="text">
      <string>100,3 Bar</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLabel" name="systemPressure">
     <property name="text">
      <string>100,0 Bar</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QLabel" name="sensor4">
     <property name="text">
      <string>100,0 Bar</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="labelP">
     <property name="text">
      <string>SystemDruck</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
